data/json/items/gun/9mm.json

```diff
@@ -402,5 +402,31 @@
       [ "46", [ "hk46mag", "hk46bigmag" ] ],
       [ "57", [ "fn57mag" ] ]
     ]
+  },
+  {
+    "id": "glock_17",
+    "copy-from": "pistol_base",
+    "type": "GUN",
+    "name": "Glock 17",
+    "description": "Designed for professionals, the Glock 17 is aimed towards law-enforcement and experienced shooters.",
```

Comment:
This is not a deal-breaker but the description is not totally correct. Yes the Glock-17 was designed for military and police use but it was also specifically designed for any type of shooter.